crates/nu-cli/src/commands/keybindings_listen.rs

```diff
@@ -20,6 +20,16 @@ impl Command for KeybindingsListen {
         "Get input from the user."
     }
 
+    fn extra_usage(&self) -> &str {
+        r#"This is an internal debugging tool. For better output, try `input listen` instead:
+
+```
+use std input
```

Comment:
actually, this is incorrect because `input listen` is a built-in command :yum:

Reply:
Oh wow. I'm glad that's the case. Teaching people about modules when they're just trying to get themselves comfortable in their shell felt a bit much.

It's a bit surprising that using a module can add a subcommand to a builtin, but I'm still getting used to this shell. I'll get there eventually.

Reply:
yeah, when you bring into scope a subcommand that has the same top command as another one in the scope, they get merged.

this is a very good thing, because you can extend the capabilities of Nushell without any extra work, e.g.
```nushell
def "from ndjson" []: string -> any {
    lines | each { from json }
}
```
will add support for the NDJSON file format, for free :yum: